### PR TITLE
Added wiki entry for Team H (Stars)

### PR DESCRIPTION
**Pull request by Harsharma2308**

### CODE REVIEW

**Comment by smbryan** [comment]
A good start, but several changes are needed.

1) Check this in a Markdown renderer. You have white space issues that will become apparent quickly.
2) You should be using `wiki/sensing` and not creating you own categorization. All assets should be in `wiki/sensing/assets` and not the top-level assets directory.
3) Missing changes to the navigation file.

**Comment by Harsharma2308** [comment]
Hi @smbryan , thanks for suggesting the changes. I have moved the entry under wiki/sensing. navigation.yml is updates as well.

File: wiki/trajectory_extraction/nearps_stereo.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import cv2
+import numpy as np
+from skimage import io
```
Comment (smbryan) [inline wiki/trajectory_extraction/nearps_stereo.py:3]: Was including this file intentional?

**Review by smbryan — COMMENTED** [review]
I'm trying to figure out how this script ties in

File: wiki/trajectory_extraction/nearps_stereo.py (hunk 1 above)
Comment (Harsharma2308) [reply wiki/trajectory_extraction/nearps_stereo.py]: no removed it

**Comment by smbryan** [comment]
Solid. Enjoy your break!

**Pull request merged**